src/QSymbolicsBase/express.jl

```diff
@@ -64,6 +64,11 @@ express_nolookup(x, repr::AbstractRepresentation, ::UseAsState) = express_nolook
 # where the cache is a distribution over possible samples.
 express_from_cache(cache) = cache
 
+function consistent_representation(regs,idx,state)
```

Comment:
`regs` is not something that QuantumSymbolics knows about. Any chance this can remain in QuantumSavory but maybe be annotated with `regs::VectorOrTupleOrWhatever{QuantumSavory.Register}`?

In particular `r.reprs` at line 68 is an implementation detail that comes from `QuantumSavory`

Reply:
seems better, for now then this PR is no longer necessary